nano/node/socket.cpp

```diff
@@ -304,14 +304,15 @@ size_t network_prefix)
 bool nano::server_socket::limit_reached_for_incoming_subnetwork_connections (std::shared_ptr<nano::socket> const & new_connection)
 {
 	debug_assert (strand.running_in_this_thread ());
-	if (node.flags.disable_max_peers_per_subnetwork)
+	if (node.flags.disable_max_peers_per_subnetwork || nano::transport::is_ipv4_or_v4_mapped_address (new_connection->remote.address ()))
```

Comment:
I do not think we need to do a source code change to handle this.
We could set the subnetwork to default to /32 (/96 for ipv6 ipv4-mapped)

Reply:
We do not have a separate subnetwork limit for ipv4 addresses, so my comment does not make any sense.